collector/analyzer/network/protocol/kafka/kafka_request_fetch.go

```diff
@@ -36,20 +34,16 @@ func parseRequestFetch() protocol.ParsePkgFn {
 			offset += 8 // session_id, session_epoch
 		}
 
-		if offset, err = message.ReadArraySize(offset, compact, &topicNum); err != nil || topicNum != 1 {
+		if offset, err = message.ReadArraySize(offset, compact, &topicNum); err != nil {
 			return false, true
 		}
-		if offset, err = message.ReadString(offset, compact, &topicName); err != nil {
-			return false, true
-		}
-		if offset, err = message.ReadArraySize(offset, compact, &partitionNum); err != nil || partitionNum != 1 {
-			return false, true
-		}
-		if _, err = message.ReadInt32(offset, &partition); err != nil {
-			return false, true
+		if topicNum > 0 {
+			if _, err = message.ReadString(offset, compact, &topicName); err != nil {
+				return false, true
+			}
+			// Read First TopicName
```

Comment:
Could you explain why this is done in comments?

Reply:
Based on following case, we	just read first topicName.

1. The payload is substr with fixed length(1K), it's not able to get all topic names
2. If we get enough length for payload(Maybe 10K), the parser will take more performance cost
3. There is not enough cases to cover multi-topics